modules/ROOT/pages/amazon/amazon-dynamodb-connector-examples.adoc

```diff
@@ -0,0 +1,700 @@
+		<http:listener-connection host="localhost" port="8081" />
+	</http:listener-config>
+	<dynamodb:config name="Amazon_DynamoDB_Configuration" doc:name="Amazon DynamoDB Configuration" >
+		<dynamodb:basic-connection accessKey="${config.accesskey}" secretKey="${config.secretkey}" />
+	</dynamodb:config>
+	<flow name="delete-table-flow" >
+		<http:listener doc:name="Listener" config-ref="HTTP_Listener_config" path="/deletetable"/>
+		<dynamodb:delete-table doc:name="Delete table" config-ref="Amazon_DynamoDB_Configuration" tableName="Issues"/>
+		<logger level="INFO" doc:name="Logger" message="#[payload]"/>
+	</flow>
+</mule>
+----
+
+=== Run the Project
+
+Save and run the project as a Mule app:
+
+. In *Package Explorer*, right-click the project and click *Run As* > *Mule Application*.
+. Open a browser and check the response after you enter the `+http://localhost:8081/deletetable+` URL.
++
+You should see the generated response from the `Delete table` operation in the Mule console.
+
+[[scan-operation]]
+== Scan an Amazon DynamoDB Table
+
+This example shows how to use Amazon DynamoDB Connector to scan an Amazon DynamoDB table.
+
+The following considerations apply to performing a scan using Amazon DynamoDB Connector:
+
+* The DynamoDB AWS SDK paginates results by dividing them into specific pages. A one MB limit applies to returned results. When this limit is exceeded, you must perform another scan to receive the rest of the data.
+* To perform subsequent scans, apply the value of the `LastEvaluatedKey` field in the JSON response to the `Exclusive start keys` field in the Studio configuration of the `Scan` operation. When the operation has returned all pages of data, the `LastEvaluatedKey` value in the JSON response is null.
+
+=== Flow for Scanning a Table
+
+The following screenshot shows an Anypoint Studio flow for scanning an Amazon DynamoDB table:
+
+.Use this sample flow to scan an Amazon DynamoDB table.
+image::amazon/amazon-dynamodb-scan-flow.png[Studio flow for scanning an Amazon DynamoDB table]
+
+=== Preliminary Steps
+
+Follow these steps to start scanning an Amazon DynamoDB table:
+
+. Create a new Mule project in Studio.
+. Add the following properties to the `mule-artifact.properties` file to hold your Amazon DynamoDB credentials:
++
+[source,text,linenums]
+----
+config.accesskey=<Access Key>
+config.secretkey=<Secret Key>
+----
++
+. Place the `mule-artifact.properties` file in the project's `src/main/resources` directory.
+
+=== Configure HTTP Listener
+
+Configure `HTTP Listener` to listen for HTTP requests on the `/scantable` path:
+
+. In the *Mule Palette*, search for `http` and select the *Listener* operation.
+. Drag the *Listener* operation onto the Studio canvas.
+. In the Listener properties window, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields for `HTTP Listener`.
```

Comment:
ditto GLOBAL re. window names. Please search for and evaluate all text around "window" per previous edits. I will stop marking.